igarss_2023/igarss_mtedgecumbe_ts_analysis.ipynb

```diff
@@ -436,10 +714,18 @@
     "!smallbaselineApp.py --dir $stack $stack/mintpy_config.txt --start quick_overview"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "2f94fe95-cd16-4bbb-bc5a-796ad587d81c",
+   "metadata": {},
+   "source": [
+    "Congrats - you have successfully completed a time series InSAR analysis! Use the plot below to view Mt. Edgecumbe's deformation history at specific locations using the interactive plot below."
```

Comment:
```suggestion
    "Congrats - you have successfully completed a time series InSAR analysis! Use the interactive plot below to view Mt. Edgecumbe's deformation history at specific locations."
```

Reply:
Maybe some guidance on how to use the plot? I can't see it, so it's not clear how intuitive that might be.